assignments/arrays.js

```diff
@@ -63,32 +74,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+function getCar(id) {
+  if (id === -1) {
+    return inventory[inventory.length - 1];
+  } else {
+    for (let i = 0; i < inventory.length; i++) {
+      if (Object.values(inventory[i])[0] === id) {
+        return inventory[i];
+      }
+    }
+  }
+}
+
+const car33 = getCar(33);
+console.log(
+  `Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}`
+);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = getCar(-1);
+console.log(
+  `Last car is a ${lastCar.car_year} ${lastCar.car_make} ${lastCar.car_model}`
+);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+function sortModels (){
```

Comment:
wow. constructing this sort function from scratch is an impressive bit of work. for future reference: JavaScript includes a built-in sort method for arrays which performs a similar logic with a slightly different syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort